airbyte-integrations/connectors/source-google-search-console/unit_tests/unit_test.py

```
@@ -123,16 +122,6 @@ def test_updated_state():
     }
 
 
-def test_forbidden_should_retry(requests_mock, forbidden_error_message_json):
```

No longer silently fail